projects/storefrontlib/shared/components/form/date-picker/date-picker.component.html

```diff
@@ -4,6 +4,7 @@
   [formControl]="control"
   [attr.min]="min"
   [attr.max]="max"
+  [value]="value"
```

Comment:
What is a purpose of such value attribute? I'm afraid that also can be a breaking change. Is it not enough to use a `formControl` value?

Reply:
That is a good point. 

We want the date picker to render a default value, if it is sent from the backend. Will reuse the `control?.value`